src/packages/auth/src/authentication/methods/utils.ts

```diff
@@ -67,3 +69,47 @@ export const runAfterHooks = async <D, H = CredentialInsertInput | CredentialUpd
 
 	return hookEntities;
 };
+
+export const makeLoginPasswordQuerySafeForLogging = (
+	queryString: string,
+	variables: VariableValues | undefined
+) => {
+	const safeVariables: VariableValues = JSON.parse(JSON.stringify(variables ?? {}));
+	const ast = parse(queryString);
```

Comment:
I think we should already have this value passed to us parsed if we're in `didResolveOperation`, no?